src/webpack5-exclude.ts

```diff
@@ -0,0 +1,24 @@
+import TestExclude from "test-exclude";
+import { defaultExclude, defaultExtensions } from "./constants";
+import { AddonOptionsWebpack } from "./types";
+import { getNycConfig } from "./nyc-config";
+
+export async function createTestExclude(
+  opts: AddonOptionsWebpack["istanbul"] = {}
+): Promise<{ shouldInstrument(filename: string): boolean }> {
+  const { nycrcPath, include, exclude, extension } = opts;
+  const cwd = opts.cwd ?? process.cwd();
+
+  const nycConfig = await getNycConfig({
+    cwd,
+    nycrcPath,
+  });
+
+  return new TestExclude({
+    cwd,
+    include: include ?? nycConfig.include,
+    exclude: exclude ?? nycConfig.exclude ?? defaultExclude,
+    extension: extension ?? nycConfig.extension ?? defaultExtensions,
```

Comment:
does this mean defaultExclude and defaultExtensions won't get merged? So if you decide to exclude `Button.*` you also lose the default exclusion of all sort of files?

Reply:
And keep in mind, that test-exclude applies some defaults internally as well: - 
- https://www.npmjs.com/package/test-exclude
- https://github.com/istanbuljs/schema/blob/master/default-exclude.js


The `defaultExclude` therefore might even not be necessary?!